build.gradle

```diff
@@ -65,8 +72,11 @@ tasks.build {
 tasks.shadowJar {
     archiveFileName = "AutoTreeChop-${version}.jar"
     configurations = [project.configurations.runtimeClasspath]
+    minimize()
 
-    relocate 'com.jeff_media.updatechecker', 'org.milkteamc.updatechecker'
+    relocate 'com.jeff_media.updatechecker', 'org.milkteamc.autotreechop.libs.updatechecker'
+    relocate 'net.kyori', 'org.milkteamc.autotreechop.libs'
+    relocate 'de.cubbossa', 'org.milkteamc.autotreechop.libs'
```

Comment:
why you need to relocate it?

Reply:
Its not necessary, but if two different plugins have the same shaded dependencies and use different versions of them, the class collisions could cause issues. The one of the two plugins whose classes were not loaded might run into issues